ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx

```diff
@@ -211,11 +212,11 @@ export function EditGroupDialogUI(props: GroupEditDialogUIProps) {
             {!group.externallyManaged && (
               <div className={classes.formActions}>
                 {isEdit && (
-                  <SecondaryButton disabled={!isDirty} onClick={onCancel}>
+                  <SecondaryButton disabled={!isDirty || inProgress} onClick={onCancel}>
                     <FormattedMessage id="words.cancel" defaultMessage="Cancel" />
                   </SecondaryButton>
                 )}
-                <PrimaryButton disabled={!submitOk} type="submit">
+                <PrimaryButton disabled={!submitOk || inProgress} loading={inProgress} type="submit">
```

Comment:
Doesn't the `loading` prop disable already?
```suggestion
                <PrimaryButton disabled={!submitOk} loading={inProgress} type="submit">
```

Reply:
it does, updated